commands/publish_dafsa.py

```diff
@@ -0,0 +1,102 @@
+import os
+import json
+import tempfile
+import subprocess
+
+import requests
+from kinto_http import Client, KintoException
+
+
+PSL_FILENAME = "public_suffix_list.dat"
+
+COMMIT_HASH_URL = (
+    f"https://api.github.com/repos/publicsuffix/list/commits?path={PSL_FILENAME}"
+)
+LIST_URL = f"https://raw.githubusercontent.com/publicsuffix/list/master/{PSL_FILENAME}"
+
+MAKE_DAFSA_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py"
+PREPARE_TLDS_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/prepare_tlds.py"
+
+BUCKET_ID = "main-workspace"
```

Comment:
maybe this one could be taken from env, because on the Dev server, since review is not enabled, we will publish to the `main` bucket directly 

```suggestion
BUCKET_ID = os.getenv("BUCKET_ID", "main-workspace")
```